tests/manager/manager-ctor-auto_encryption-001.phpt

```diff
@@ -0,0 +1,83 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): auto encryption options
+--FILE--
+<?php
+
+$baseOptions = [
+    'keyVaultNamespace' => 'admin.dataKeys',
+    'kmsProviders' => ['aws' => (object) ['accessKeyId' => 'abc', 'secretAccessKey' => 'def']]
+];
+
+$tests = [
+    [],
+    ['keyVaultClient' => new MongoDB\Driver\Manager()],
+    ['schemaMap' => [
+        'default.default' => [
+            'properties' => [
+                'encrypted_objectId' => [
+                    'encrypt' => [
+                        'keyId' => [
+                            [
+                                '$binary' => [
+                                    'base64' => 'AAAAAAAAAAAAAAAAAAAAAA==',
+                                    'subType' => '04',
+                                ],
+                            ],
+                        ],
+                        'bsonType' => 'objectId',
+                        'algorithm' => 'AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic',
+                    ],
+                ],
+            ],
+            'bsonType' => 'object',
+        ],
+    ]],
+    ['bypassAutoEncryption' => true],
+    ['extraOptions' => ['mongocryptdBypassSpawn' => true]],
+];
+
+foreach ($tests as $driverOptions) {
```

Comment:
I think `$autoEncryptionOptions` would be a better name, as the values in `$tests` above are all options that go _within_ the `autoEncryption` key.

`$driverOptions` would be the third argument to the Manager constructor, so that name would make more sense if each value in `$tests` started with an `autoEncryption` key.